content/contributing-terms.en.md

```diff
@@ -389,6 +389,22 @@ If some parts of the source documents are repeated, they can be factorised. For
   …
 ```
 
+## Contributing new declarations
+
+This is a step by step guide to help you add declarations to the [contrib-declaration](https://github.com/OpenTermsArchive/contrib-declarations) repository. This repository is dedicated for volunteer contribution of declarations to Open Terms Archive.
```

Comment:
While these are technically repositories, let's use the business term of “collection” as it describes more precisely what these repositories are intended for 🙂 

```suggestion
This is a step by step guide to help you add declarations to the [Contrib collection](https://github.com/OpenTermsArchive/contrib-declarations). This collection is dedicated to volunteer contribution of declarations to Open Terms Archive.
```